cli/internal/cmd/upgradeapply.go

```diff
@@ -551,6 +593,20 @@ type upgradeApplyFlags struct {
 	terraformLogLevel terraform.LogLevel
 	conformance       bool
 	helmWaitMode      helm.WaitMode
+	SkipPhases        SkipPhases
+}
+
+// SkipPhases is a list of phases that can be skipped during the upgrade process.
+type SkipPhases []SkipPhase
+
+// Contains returns true if the list of phases contains the given phase.
+func (s SkipPhases) Contains(phase SkipPhase) bool {
+	for _, p := range s {
+		if p == phase {
```

Comment:
```suggestion
		if strings.EqualFold(p, phase) {
```
Use [`strings.EqualFold`](https://pkg.go.dev/strings#EqualFold) to have a reliable way for comparing case-insensitive strings.